src/ImportCsvDialog.cpp

```diff
@@ -412,6 +449,55 @@ void ImportCsvDialog::importCsv(const QString& fileName)
     }
 }
 
+sqlb::FieldVector ImportCsvDialog::fetchCsvHeader(const QString& fileName)
```

Comment:
> Though I'm not exactly sure how to do that ideally...

Had the same thought then that :), it was a bit tricky seeing how the FieldVecs and csv object are used because I didn't want to upset or refactor too much code initially but I'll come up with something.